src/shared/components/Combobox.tsx

```diff
@@ -49,9 +56,15 @@ export default function Combobox(props: Props) {
   };
   const hide = () => {
     setOpen(false);
-    onHide();
+    // onHide();
   };
 
+  /* here could be some type checking is the itemSource an array etc.
+     Also in case of complicated data this mapper should be placed in the different file
+     in the mapper folder and it should map source to the Entities. like new SelectedItem() etc.
+  */
+  const options = itemsSource.map(itemName => ({ value: itemName.toLowerCase(), label: itemName }));
```

Comment:
I think this will be simpler if the options is simple an array of combobox items where the value points to the original object.  It is possible that two items might have the same name and we don't want that to cause a collision.

Reply:
hmmm, but what is the reason to let the users choose one from two the same options in the combobox? I don't understand
Btw, we definitely need some mapper because ReactSelect works with that type: { label : <val>, value : <val> }
![image](https://user-images.githubusercontent.com/9897782/69912210-a2c39800-142e-11ea-98e5-c86f52aacb17.png)

Reply:
General purpose comboboxes cannot assume that items will have unique string representations.  (Well they could, but in practice this creates more problems than it solves)